swagger.yml

```
@@ -5113,32 +5165,35 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/PartMetadata'
-        payment_options:
-          type: array
-          items:
-            $ref: '#/components/schemas/PartnerPaymentOption'
-        references:
-          $ref: '#/components/schemas/InvoiceReferences'
         received_at:
+          description: When the content was received by Kivra
           type: string
           format: date-time
         receiver_name:
+          description: The name of the recipient company
           type: string
         receiver_vat_number:
           $ref: '#/components/schemas/VatNumber'
         sender_icon_url:
           type: string
           example: "https://sandbox-static.kivra.com/img/tenant/1341573157b3a1c55f220f4007b2e44789d2d04015/icon.png"
           format: url
-          description: An url to the icon used by the sender within Kivra, or `null` if no corresponding icon exists.
+          description: A url to the icon used by the sender within Kivra, or `null` if no corresponding icon exists.
           nullable: true
         sender_name:
+          description: The name of the sender
+          example: Lenas Konditori AB
           type: string
+          nullable: true
         sender_vat_numbers:
+          description: VAT numbers representing the sender
           type: array
           items:
             $ref: '#/components/schemas/VatNumber'
+          nullable: true
         subject:
+          description: Subject of the content
+          example: "Faktura: tusen rulltårtor"
```

Would be nicer to keep the examples in English.
```suggestion
          example: "Invoice: a thousand Swiss rolls"
```
:laughing: